File: app/src/main/java/com/callmangement/EHR/models/DealersInfo.java
package com.callmangement.EHR.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class DealersInfo implements Serializable {
    @SerializedName("customerNameEng")
    private String customerNameEng;

    @SerializedName("customerNameHi")
    private String customerNameHi;

    @SerializedName("fpscode")
    private String fpscode;

    @SerializedName("mobileNo")
    private String mobileNo;

    public String getCustomerNameEng() {
        return customerNameEng;
    }

    public void setCustomerNameEng(String customerNameEng) {
        this.customerNameEng = customerNameEng;
    }

    public String getCustomerNameHi() {
        return customerNameHi;
    }

    public void setCustomerNameHi(String customerNameHi) {
        this.customerNameHi = customerNameHi;
    }

    public String getFpscode() {
        return fpscode;
    }

    public void setFpscode(String fpscode) {
        this.fpscode = fpscode;
    }

    public String getMobileNo() {
        return mobileNo;
    }

    public void setMobileNo(String mobileNo) {
        this.mobileNo = mobileNo;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/DealersInfo.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class DealersInfo : Serializable {
    @JvmField
    @SerializedName("customerNameEng")
    var customerNameEng: String? = null

    @SerializedName("customerNameHi")
    var customerNameHi: String? = null

    @JvmField
    @SerializedName("fpscode")
    var fpscode: String? = null

    @JvmField
    @SerializedName("mobileNo")
    var mobileNo: String? = null
}

